tipocket-ctl/tpctl/scripts/env_raw.sh

```diff
@@ -9,13 +9,27 @@ CLUSTER_NAMESPACE=$(argo get -n argo $DEPLOY_ID -o json \
                     | grep -o '".*"' \
                     | sed -e 's/^"//' -e 's/"$//')
 
+function t_get_grafana_port {
+	grafana_port=$(kubectl -n $CLUSTER_NAMESPACE get svc -o json \
+		       | jq -r '.items[].spec.ports[]
+			        | select(.name == "http-grafana")
+				| .nodePort')
+	if [ "$grafana_port" == "" ]; then
+		echo "Grafana is not ready yet, please run t_get_grafana_port later."
+	else
+		echo "Grafana is on port $grafana_port, username and password are both admin."
```

Comment:
How about printing a whole http url like `http://172.x.y.z:{port}`

Reply:
And we can also show the TiDB addr like `mysql -uroot -P{port} -h 172.x.y.z`

Reply:
Done.

Reply:
Done.